lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2435,18 +2440,18 @@ def bar3d(self, x, y, z, dx, dy, dz, color=None,
 
         zsort : str, optional
             The z-axis sorting scheme passed onto
-            :func:`~mpl_toolkits.mplot3d.art3d.Poly3DCollection`
+            :class:`~.art3d.Poly3DCollection`
 
         shade : bool, optional (default = True)
             When true, this shades the dark sides of the bars (relative
             to the plot's source of light).
 
-        Any additional keyword arguments are passed onto
-        :func:`~mpl_toolkits.mplot3d.art3d.Poly3DCollection`
+        **kwargs : Any additional keyword arguments are passed onto
+            :class:`~.art3d.Poly3DCollection`.
```

Comment:
`:class:` see above. Also, I would move the sentence to the description.

~~~
**kwargs
    Any additional keyword arguments are passed onto
    `~.art3d.Poly3DCollection`.
~~~